### PR TITLE
click of node should select/unselect the node

### PR DESCRIPTION
**Pull request by srilathass**
## PR Checklist
click of a node should select/unselect the node. Related ticket # https://mindbridge-ai.atlassian.net/browse/AIA-13921

- [ ] The commit message follows our guidelines: https://github.com/CirclonGroup/angular-tree-component/blob/master/CONTRIBUTING.md#commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/angular-tree-component/src/lib/models/tree-options.model.ts [hunk 1]
```diff
@@ -37,7 +54,7 @@ export const TREE_ACTIONS = {
 
 const defaultActionMapping: IActionMapping = {
   mouse: {
-    click: TREE_ACTIONS.TOGGLE_ACTIVE,
+    click: TREE_ACTIONS.TOGGLE_SELECTED,
```
Comment (srilathass) [inline projects/angular-tree-component/src/lib/models/tree-options.model.ts:57]: This is the only change I did and other changes are saved along with this

Comment (shardyMBAI) [reply projects/angular-tree-component/src/lib/models/tree-options.model.ts:57]: I think you might have global save on format or something, I am sorry about this, but can we try to keep the code in the style for this project (as the formatting changes seem mostly to follow AIA). @stevensoncam might be able to help as he recently edited this project as well.

**Review by shardyMBAI — COMMENTED** [review]
Some formatting changes might not be needed, but some look like they are? I think it's best to follow the projects setup.

Unfortunately, I am not too familiar with this project or its set up, so I'll deffer to @stevensoncam opinion.

File: projects/angular-tree-component/src/lib/models/tree-options.model.ts (hunk 1 above)
Comment (srilathass) [reply projects/angular-tree-component/src/lib/models/tree-options.model.ts:57]: these changes are added on PreCommit step

Comment (stevensoncam) [reply projects/angular-tree-component/src/lib/models/tree-options.model.ts:57]: Maybe you edited the file from the same VSCode instance as our app? I started a new instance with code . in the angular-tree-component directory

Comment (srilathass) [reply projects/angular-tree-component/src/lib/models/tree-options.model.ts:57]: when I tried from IntelliJ IDE, same changes got added up at pre-commit step.

Comment (srilathass) [reply projects/angular-tree-component/src/lib/models/tree-options.model.ts:40]: I did git commit -n to avoid those hooks and it worked.

Comment (shardyMBAI) [reply projects/angular-tree-component/src/lib/models/tree-options.model.ts:40]: nice! thank you.

**Review by stevensoncam — APPROVED** [review]

**Pull request merged**